### ISSUE TITLE
Provider to help with JIT client definition

### ISSUE THREAD

**Issue by luckymike** [sensu/sensu-chef#417]
Would it be helpful to have a provider to define JIT clients that execute a remote or repeated check?

**Comment by runningman84** [comment]
Yes that would be really helpful for us. We would like to define jit clients for aws elb and rds services.

**Comment by Evesy** [comment]
I too would find this useful to have. Especially with the recent introduction of proxy requests, there is now a bigger need for a simple method to add clients to the registry.

**Comment by majormoses** [comment]
Pretty sure this is actually trivial as all that needs to be defined is source right? If so I can do this pretty easily as it just means adding it to: https://github.com/sensu/sensu-chef/blob/v4.1.0/providers/check.rb#L16-L19 I will put together a PR and link it. If there is anything else let me know.

**Comment by majormoses** [comment]
See the linked PR, if there is more than just source that needs to be added just let me know.
